tests/parser.test.js

```diff
@@ -0,0 +1,12 @@
+const tokenize = require('../src/tokenizer.js').tokenize;
```

Comment:
Empty test suites should not be included. They cause failures when running tests.